### ISSUE TITLE
关于 dense relation distill module

### ISSUE THREAD

**Issue by Saralyliu** [hzhupku/DCNet#25]
def forward(self,features,attentions):
        features = list(features)
        if isinstance(attentions,dict):
            for i in range(len(attentions)):
                if i==0:
                    atten = attentions[i].unsqueeze(0)
                else:
                    atten = torch.cat((atten,attentions[i].unsqueeze(0)),dim=0)
            attentions = atten.cuda()
        output = []
        h , w = attentions.shape[2:]
        ncls = attentions.shape[0]       
        key_t = self.key_t(attentions)   
        val_t = self.value_t(attentions) 
        for idx in range(len(features)):

我想知道谁是support ，谁是query？按代码的意思，是attentions是support，features是query是这样吗？
attentions和features的维度分别是多少？
期待您的回复，谢谢！
